### DISCUSSION TITLE
从0.11.2版本升级到0.12后，原来的0.11.2版本的数据库如何迁移到0.12版本中？

### DISCUSSION THREAD

**Discussion by ttwxz** [usememos/memos#1473] [General]
memos升级算是很频繁的，从0.10版本升级到0.11.2版本很顺利，数据库映射原来的，升级成功，但是从0.11.2升级到0.12时，创建数据库经常出错，安装了很多次，均失败了，后来没办法，只能重装0.12的版本，但原来的数据如何迁移到0.12版本中？
请问：1，原来0.11.2版本的数据如何迁移到0.12版本中？
           2.如何在保留数据库的情况下升级？

**Comment by villen8** [comment]
是吖，如何保存数据更新导入？iStoreOS中的Docker按了更新后，虽然更新到了最新版本，但原来的数据也没
